docs/sbt.md

```diff
@@ -0,0 +1,275 @@
+---
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+
+# Building via SBT
+
+Starting from version 0.4.0, the Celeborn project supports building and packaging using SBT. This article provides a detailed guide on how to build the Celeborn project using SBT.
+
+# System Requirements
+
+Celeborn Service (master/worker) supports Scala 2.11/Scala 2.12 and Java 8/11/17.
+
+The table following indicates the compatibility of Celeborn Spark and Flink clients with different versions of Spark and Flink for various Java and Scala versions:
+
+|                | Java 8/Scala 2.11 | Java 8/Scala 2.12 | Java 11/Scala 2.12 | Java 17/Scala 2.12 |
+|----------------|-------------------|-------------------|--------------------|--------------------|
+| Spark 2.4      | &#10004;          | &#x274C;          | &#x274C;           | &#x274C;           |
+| Spark 3.0      | &#x274C;          | &#10004;          | &#10004;           | &#x274C;           |
+| Spark 3.1      | &#x274C;          | &#10004;          | &#10004;           | &#x274C;           |
+| Spark 3.2      | &#x274C;          | &#10004;          | &#10004;           | &#x274C;           |
+| Spark 3.3      | &#x274C;          | &#10004;          | &#10004;           | &#10004;           |
+| Spark 3.4      | &#x274C;          | &#10004;          | &#10004;           | &#10004;           |
+| Flink 1.14     | &#x274C;          | &#10004;          | &#10004;           | &#x274C;           |
+| Flink 1.15     | &#x274C;          | &#10004;          | &#10004;           | &#x274C;           |
+| Flink 1.17     | &#x274C;          | &#10004;          | &#10004;           | &#x274C;           |
+
+
+# Setting up Private Repository Proxy
+
+If you don't encounter any network issues, you can skip this step.
+
+## Setting up Repositories for Chinese Developers
+
+As a developer in China, you might face slow resource loading during the sbt startup process. To accelerate resource retrieval, you can replace the default repositories with the following configurations.
```

Comment:
not only china mainland, there are many areas that may face low speed issues, it's a generic technical issue. the thing we can do is to provide area-specific `repositories-<area>.template`s with verified mirror.